packages/legend-extension-dsl-data-space/src/stores/query/DataSpaceQueryCreatorStore.ts

```diff
@@ -109,6 +109,10 @@ export class DataSpaceQueryCreatorStore extends QueryEditorStore {
     };
   }
 
+  override get label(): string {
+    return this.dataSpacePath;
```

Comment:
This label won't help user neither, maybe something like `Data Space: ${dataSpace.title/name}` or something?

Reply:
or something suggest that we're creating something new... ?